app/controllers/tracks/exercises_controller.rb

```diff
@@ -44,6 +44,9 @@ def completed
     else
       @concepts = @exercise.prerequisites
     end
+
+    @unlocked_exercises = @track.exercises.limit(2)
```

Comment:
Is this temporary? As in: do you only ever want to show two? Or if this is temporary, maybe add a TODO?

Reply:
Yeah, this whole method has a TODO to delete it - it's just not shown in the diff. These methods/templates are just here for Karlo to reimplement in React basically. The actual server-side stuff goes via the API.

Good catch though! Thank you :)